### PR TITLE
Refactor Overview, StaticAppBar#1697

### LINKED ISSUE #1697: Refactor Overview.react.js, StaticAppBar.react.js
**Actual Behaviour**

Overview.react.js, StaticAppBar.react.js needs to be consistent with codebase

**Expected Behaviour**

Refactor Overview.react.js, StaticAppBar.react.js

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by fragm3**
Fixes #1697 

Changes: 

- Ordered import statements
- constant declared before components to make it consistent
- Used arrow functions
- Made let to const for read only
- **Overview.React.js**: Remove login, signup unwanted state, handleClose method
- **StaticAppBar.js**: baseUrl changed to constant from state, remove closingStyleLogin style redundancy

Demo Link: https://pr-1697-fossasia-susi-web-chat.surge.sh

### CODE REVIEW

File: src/components/Overview/Overview.react.js [hunk 1]
```diff
@@ -31,31 +44,17 @@ class Overview extends Component {
   handleTitle = () => {
     this.props.history.push('/');
   };
-  // Open Video Dialog and close login and sign up dialog
+  // Open Video dialog
   handleVideo = () =>
```
Comment (akshatnitd) [inline src/components/Overview/Overview.react.js]: Rename it to `handleVideoModalOpen`

Comment (akshatnitd) [reply src/components/Overview/Overview.react.js]: Instead make a common component for handling the open/close state of Video Modal and name it `toggleVideoModal`

File: src/components/StaticAppBar/StaticAppBar.react.js [hunk 2]
```diff
@@ -230,7 +290,14 @@ class StaticAppBar extends Component {
...
+
+    setInterval(() => {
+      if (didScroll) {
+        hasScrolled();
+        didScroll = false;
+      }
+    }, 500);
 
     // Return menu items for the hamburger menu
     Logged = props => (
```
Comment (akshatnitd) [inline src/components/StaticAppBar/StaticAppBar.react.js]: Change the function name to `renderMenuItems`

**Comment by fragm3** [comment]
@akshatnitd thanks for the review, made the changes!

File: src/components/StaticAppBar/StaticAppBar.react.js (hunk 2 above)
Comment (fragm3) [reply src/components/StaticAppBar/StaticAppBar.react.js]: By changing the name, its not working.

File: src/components/Overview/Overview.react.js [hunk 3]
```diff
@@ -27,34 +40,28 @@ class Overview extends Component {
       video: false,
     };
   }
-
-  handleTitle = () => {
-    this.props.history.push('/');
+  // Toggle Video dialog
+  toggleVideoModal = () => {
+    if (this.state.video === true) {
```
Comment (akshatnitd) [inline src/components/Overview/Overview.react.js]: Use - 
```
this.setState((prevState) => ({
  isVideoModalOpen: !prevState.isVideoModalOpen
}));
```

Comment (akshatnitd) [reply src/components/Overview/Overview.react.js]: Also rename the state to `isVideoModalOpen`

File: src/components/Overview/Overview.react.js [hunk 4]
```diff
@@ -27,34 +40,28 @@ class Overview extends Component {
...
-  closeVideo = () =>
-    this.setState({
-      video: false,
-    });
 
-  _onReady(event) {
-    // access to player in all event handlers via event.target
-    event.target.pauseVideo();
-  }
+  // handleVideo = () =>
```
Comment (akshatnitd) [inline src/components/Overview/Overview.react.js]: Remove comments

File: src/components/Overview/Overview.react.js [hunk 5]
```diff
@@ -27,34 +40,28 @@ class Overview extends Component {
...
-      signup: false,
-      video: false,
-    });
-  // Close Video Dialog
-  closeVideo = () =>
-    this.setState({
-      video: false,
-    });
 
-  _onReady(event) {
```
Comment (akshatnitd) [inline src/components/Overview/Overview.react.js]: Is is not required?

Comment (fragm3) [reply src/components/Overview/Overview.react.js]: Yes not required. So I removed it.

**Pull request closed without merging**